app/controllers/angular_template_controller.rb

```diff
@@ -0,0 +1,5 @@
+# used to serve static angular templates from under app/views/angular_template/
+
+class AngularTemplateController < ApplicationController
```

Comment:
@abonas , this is not meant to be extended nor included, it's simply a mechanism to provide raw html templates to angular directives (or ) using `templateUrl`. It basically exists so that you don't have to manually provide routes for things that should be just downloadable files. You don't use it from Rails side, you use it by knowing that `GET /angular_template/whatever` maps  to `app/views/angular_template/whatever.html` (or `whatever.html.haml`).